must-gather/collection-scripts/pre-install.sh

```diff
@@ -58,6 +58,22 @@ check_for_helper_pod(){
     oc wait -n openshift-storage --for=condition=Ready pod/"${HOSTNAME}"-helper --timeout=200s
 }
 
+cleanup() {
+  echo "checking for existing must-gather resource" | tee -a "${BASE_COLLECTION_PATH}"/gather-debug.log
+  pods=$(oc get pods --no-headers -n openshift-storage -l must-gather-helper-pod='' | awk '{print $1}')
+  if [ -n "${storageClusterPresent}" ] && [ -n "${pods}" ]; then
+    SAVEIFS=$IFS # Save current IFS
+    IFS=$'\n'    # Change IFS to new line
+    pods=($pods) # split to array $pods
+    IFS=$SAVEIFS # Restore IFS
+    echo "deleting existing must-gather resource" | tee -a "${BASE_COLLECTION_PATH}"/gather-debug.log
+    for pod in "${pods[@]}"; do
+      oc delete pod "${pod}" -n "${ns}"
```

Comment:
Why don't we use the same logic which is there in post-install script

```
if [ -n "$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')" ]; then
    oc delete -f pod_helper.yaml
fi

```

Reply:
The pod template is updated in 
`apply_helper_pod() {
    < ${POD_TEMPLATE} safe_replace "NAMESPACE" "$1" | safe_replace "IMAGE_NAME" "$2" | safe_replace "MUST_GATHER" "$HOSTNAME" > pod_helper.yaml
    oc apply -f pod_helper.yaml
}`
will the variables in pod_helper.yaml be the same when we run a different instance of must-gather? If that is true then we can call the post-install script here and resue the script

Reply:
Not sure, can you please check once.

Reply:
I will try running the clean up with post-install script and see how it works

Reply:
Ok, thanks for checking. Lets go ahead with this.